hazelcast-enterprise-openshift-centos/Dockerfile

```diff
@@ -1,21 +1,9 @@
-FROM centos:7
-MAINTAINER Hazelcast, Inc. Integration Team <info@hazelcast.com>
-
-ENV HZ_HOME /opt/hazelcast/
-ENV HZ_CP_MOUNT ${HZ_HOME}/external
-ENV JAVA_HOME /usr/lib/jvm/java
-ENV LANG en_US.utf8
-
-ENV USER_NAME=hazelcast
-ENV USER_UID=10001
+ARG HZ_IMAGE_TAG=latest
```

Comment:
I am not sure using latest here is a good choice because github tag will not include exact released version.